adapter/internal/operator/controllers/dp/api_controller.go

```diff
@@ -315,6 +328,36 @@ func (apiReconciler *APIReconciler) getHostNameForBackend(ctx context.Context, b
 	return utils.GetDefaultHostNameForBackend(backend, defaultNamespace)
 }
 
+// getTLSConfigForBackend resolves backend TLS configurations.
+func (apiReconciler *APIReconciler) getTLSConfigForBackend(ctx context.Context,
+	serviceNamespacedName types.NamespacedName) dpv1alpha1.TLSConfigs {
+	tlsConfig := dpv1alpha1.TLSConfigs{}
+	backendPolicyList := &dpv1alpha1.BackendPolicyList{}
+	if err := apiReconciler.client.List(ctx, backendPolicyList, &k8client.ListOptions{
+		FieldSelector: fields.OneTermEqualSelector(serviceBackendPolicy, serviceNamespacedName.String()),
+	}); err != nil {
+		loggers.LoggerAPKOperator.ErrorC(logging.ErrorDetails{
+			Message:   fmt.Sprintf("Unable to find associated BackendPolicies for service: %s", serviceNamespacedName),
+			Severity:  logging.CRITICAL,
+			ErrorCode: 2611,
+		})
+	}
+	if len(backendPolicyList.Items) > 0 {
+		backendPolicy := backendPolicyList.Items[0]
+		if len(backendPolicyList.Items) > 1 {
```

Comment:
mmm... we can't just ignore other policies. you need to follow policy spec guideline for handling this. // According to https://gateway-api.sigs.k8s.io/geps/gep-713/?h=multiple+targetrefs#conflict-resolution. Refer selectAuthScheme in http_route.go